app/modules/users/components/lib/navs_helpers.js

```diff
@@ -0,0 +1,14 @@
+import User from '#users/models/user'
+import Group from '#groups/models/group'
+import { isOpenedOutside } from '#lib/utils'
+
+export const isNotMainUser = user => user._id !== app.user.id
+
+export const SelectInventoryOnClick = ({ type, doc }) => e => {
+  if (isOpenedOutside(e)) return
+  let model
+  if (type === 'user') model = new User(doc)
+  else if (type === 'group') model = new Group(doc)
+  app.vent.trigger('inventory:select', type, model)
```

Comment:
When i click on a user marker, it scrolls to the top tabs.
But when i click on a group marker it scrolls to the inventory

Reply:
might have been fixed by 0a938b0